content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx

```diff
@@ -56,11 +56,15 @@ In other development environments (for example, other team members' computers):
 
 ## Baseline your production environment
 
-Assuming you have successfully transitioned to the new version of Prisma Migrate and initialized a new migration history in your repository, you now need to **baseline** any database that contains important data - such as production or staging.
+Baselining is for introducing Prisma Migrate to existing databases / brownfield projects. Concretely it's to tell Prisma Migrate to assume that one or more migration(s) were already executed so it won't try to drop and recreate existing tables.
 
-In development, we assume it is acceptable to reset your database to make sure your migration files are in sync with your database schema. This is not acceptable in a production environment, or any other environment where data must be maintained.
+Assuming you started a new migration history in your repository using `prisma migrate dev`, you now need to **baseline** any database that contains important data – such as production or staging – where the schema was not created using the `prisma migrate deploy` command.
```

Comment:
I think "where the schema was not created using the `prisma migrate deploy` command" could be a bit confusing to users. Another way to formulate _when  you should baseline is "the first time you want to `prisma migrate deploy` to your existing database".

Reply:
I agree with you :)

```suggestion
Assuming you started a new migration history in your repository using `prisma migrate dev`, you now need to **baseline** any database that contains important data – such as production or staging – the first time you want to `prisma migrate deploy` to your existing database.
```